sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobBeginCopyOptions.java

```diff
@@ -160,7 +160,7 @@ public BlobBeginCopyOptions setPollInterval(Duration pollInterval) {
 
     /**
      *  Only applicable for Append Blobs.
-     * @return Whether or not the destination blob should be sealed.
+     * @return Whether or not the destination blob should be sealed (marked as read only).
```

Comment:
Is there a reason we left out `This is only applicable for Append blobs.` on this instance?

Reply:
It was already in the line above (162). I can bring it down if you want.

Reply:
Nah I'm just blind.